core/models/model_training.py

```diff
@@ -0,0 +1,113 @@
+import torch
+import copy
+import sys
+from tqdm.notebook import tqdm
+from torch.utils.data import DataLoader
+import torch.nn as nn
+from core.util.early_stop import EarlyStop
+
+learning_rate = 0.001
```

Comment:
Why is this parameter defined in this scope, if it is only one function that uses it?